contracts/IdentityOracle.sol

```diff
@@ -12,19 +12,15 @@ contract IdentityOracle is DAOUpgradeableContract, ChainlinkClient, Ownable {
     address public dao_avatar; // this implementation is only to test. In live it would be replaced for dao.avatar
 
     bytes32 public stateHash; // current state hash
-    string public stateDataIPFS; // ipfs cid
-
-    bytes32 public tmpStateHash; // this is used for the first of three steps of oracle updating process
-    bytes32 public tmpStateDataIPFS1; // this is used for the second of three steps of oracle updating process
-    bytes32 public tmpStateDataIPFS2; // this is used for the second of three steps of oracle updating process
-
-    uint256 private constant ORACLE_PAYMENT = 10**17;
-    string private JOBID_GET_STATE_HASH = "9088a872ef6a4c9b98234ac8db3ec4c0";
-    string private JOBID_GET_IPFS_CID1 = "abbcfd8e16c048718e8a030daa2e489b";
-    string private JOBID_GET_IPFS_CID2 = "be8deaa69e5b4dbea17f20fbc364023b";
-    address private constant ORACLE_ADDRESS =
-        0x7bccD524Fd2Cc264Dd80E4589B4B3409fb61daC0;
-
+    string  public stateDataIPFS; // ipfs cid
+
+    uint256 private constant ORACLE_PAYMENT = 10**16;
+    string  private JOBID_GET_STATE_HASH_IPFSCID = "a36cda0a7636428db24b1e61a5a36976";
```

Comment:
whats that?

Reply:
you  can make all variables public, easier to debug contract when it is deployed and make it more transparent

Reply:
fixed by https://github.com/GoodDollar/ChainlinkIdentityOracle/pull/5/commits/f05ce99726a869f1dfaf944456647138eaf25fa3